crypto/ocsp/ocsp_extension.c

```diff
@@ -37,6 +37,19 @@ X509_EXTENSION *OCSP_BASICRESP_get_ext(OCSP_BASICRESP *bs, int loc) {
   return X509v3_get_ext(bs->tbsResponseData->responseExtensions, loc);
 }
 
+int OCSP_SINGLERESP_add_ext(OCSP_SINGLERESP *sresp, X509_EXTENSION *ex,
```

Comment:
Missing null checks.

Reply:
Yeah I was on the fence for these, they're single calls against existing functions that already have null checks. 
https://github.com/aws/aws-lc/blob/34aceaf04b1ff0499104a73e67c7d4b36551f1f8/crypto/x509/x509_v3.c#L67-L124

Should we add them here? I'm ok either way.

Reply:
if there's no way for a NULL pointer passed in by the caller to segfault, then IMO we can skip redundant null checks.